src/styles/style.scss

```diff
@@ -0,0 +1,134 @@
+$card-height: 200px;
+$container-padding: 0 16px 16px;
+$default-font-size: 10px;
+$stars-numbers: 4;
+$main-color: #060b35;
+$secondary-color: #616070;
+$color-button: #00acdc;
+
+h1,
+h2,
+h3,
+p {
+  margin: 0;
+  padding: 0;
+}
+
+body {
+  margin: 0;
+  padding: 0;
+  height: 100vh;
+  background-color: #aaa;
+  color: $main-color;
+  font-family: "Roboto", Arial, Helvetica, sans-serif;
+  display: flex;
+  align-items: center;
+  justify-content: center;
+}
+
+.card {
+  box-sizing: border-box;
+  width: $card-height;
+  border: 1px solid #f3f3f3;
+  border-radius: 5px;
+  background-color: #fff;
+
+  &__img {
+    display: block;
+    height: 134px;
+    margin: 32px auto 40px;
+  }
+
+  &__container {
+    padding: $container-padding;
+  }
+
+  &__title {
+    font-size: 12px;
+    line-height: 18px;
+    font-weight: 500;
+  }
+
+  &__product-code {
+    margin-top: 4px;
+    font-size: $default-font-size;
+    line-height: 14px;
+    font-weight: 400;
+    color: $secondary-color;
+  }
+
+  &__rating {
+    display: flex;
+    align-items: flex-end;
+    justify-content: space-between;
+    margin-top: 16px;
+  }
+
+  &__reviews {
+    font-size: $default-font-size;
+    line-height: 14px;
+    font-weight: 400;
+  }
+
+  &__price {
+    display: flex;
+    align-items: center;
+    justify-content: space-between;
+    margin-top: 24px;
+  }
+
+  &__text {
+    color: $secondary-color;
+    font-size: 12px;
+    line-height: 18px;
+    font-weight: 400;
+  }
+
+  &__value {
+    font-size: 16px;
+    line-height: 18px;
+    font-weight: 700;
+  }
+
+  &__button {
+    display: flex;
+    align-items: center;
+    justify-content: center;
+    background-color: $color-button;
+    border: 1px solid $color-button;
+    border-radius: 5px;
+    margin-top: 16px;
+    width: 166px;
+    height: 40px;
+    text-transform: uppercase;
+    text-decoration: none;
+    color: #fff;
+    font-size: 14px;
+    font-weight: 700;
+  }
+
+  &__button:hover {
+    background-color: #fff;
+    color: #00acdc;
+  }
+}
+
+.stars {
+  display: flex;
+
+  &__star {
+    background-image: url(../images/star.svg);
+    background-repeat: no-repeat;
+    background-position: center;
+    width: 16px;
+    height: 16px;
+
+    &:not(:first-child) {
+      margin-left: 4px;
+    }
+  }
+
+  &--4 &__star:nth-child(-n + #{$stars-numbers}) {
```

Comment:
please add all needed styles from the previous task
![image](https://github.com/mate-academy/layout_product-cards/assets/22824431/5210997d-c774-4fbe-8d5a-6a639b175613)